spec/models/dor_object_workflow_status_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe DorObjectWorkflowStatus do
+  subject { described_class.new(pid) }
+  let(:pid) { 'druid:abc123' }
+  describe '#can_open_version?' do
+    context 'when not accessioned' do
+      before do
+        expect(Dor::WorkflowService)
+          .to receive(:get_lifecycle).with('dor', pid, 'accessioned')
+          .and_return(false)
+      end
+      it { expect(subject.can_open_version?).to eq false }
+    end
```

Comment:
I guess I disagree here. I think contexts are useful in this case. If the spec fails you see the output as:

"DorObjectWorkflowStatus#can_open_version? when accessioned and submitted should eq true"

If it is switched to an `it` block, you see:

"DorObjectWorkflowStatus#can_open_version? when accessioned and submitted"

http://betterspecs.org/#contexts